build/Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM centos:centos8
+
+ARG CHROMEDRIVER_VER=90.0.4430.24
```

Comment:
I think doing something like this would be better:

```
RUN CHROMEVER=$(google-chrome-stable --product-version | grep -o "[^\.]*\.[^\.]*\.[^\.]*") && \
         DRIVERVER=$(curl -s "https://chromedriver.storage.googleapis.com/LATEST_RELEASE_$CHROMEVER") && \
         wget -q --continue -P /chromedriver "http://chromedriver.storage.googleapis.com/$DRIVERVER/chromedriver_linux64.zip" && \
         unzip /chromedriver/chromedriver* -d /chromedriver
```

Reply:
chromedriver 89.x.x.x seems to be [present in the centos epel repository,](https://pkgs.org/search/?q=chromedriver) so if this version suits your needs you could just run `dnf install chromedriver`.

Reply:
That would simplify the setup but we would have to switch to the `chromium-browser` to use the `chromedriver` package in the epel repo since it appears that those two versions are synchronized for updates.

Reply:
chrome and chromium are mostly the same so if it simplifies our testing setup I'll go for it if nobody have any concern

Reply:
Updated to pull `chromium-browser` & `chromedriver` from the `epel` repo